packages/radio-player/src/radio-player.ts

```diff
@@ -274,15 +274,15 @@ export default class RadioPlayer extends LitElement {
    * @memberof RadioPlayer
    */
   private get waveFormProgressTemplate(): TemplateResult {
-    return html`
+    return this.waveformUrl ? html`
       <waveform-progress
         interactive="true"
         .waveformUrl=${this.waveformUrl}
         .percentComplete=${this.percentComplete}
         @valuechange=${this.valueChangedFromScrub}
       >
       </waveform-progress>
-    `;
+    ` : html``;
```

Comment:
is this the way we should be returning "nothing"? In React, we return a `null` and it doesn't add excess markup to actual DOM.

Reply:
I changed it to return `undefined` if there's no template. That does the trick! Good call.